app/points/calculate-sdr-conversion-points.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (SDRConversionsLast30Days, WorkloadPointsSDRConversion) {
```

Comment:
Had just noticed this wasn't working for PAROMS calculation and changed it haha. Changing now, cheers 👍